modules/ossm-operatorhub-remove.adoc

```diff
@@ -0,0 +1,121 @@
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
+[id="ossm-remove-operator-elasticsearch_{context}"]
+== Removing the Elasticsearch Operator
+Follow this procedure to remove the Elasticsearch Operator.
+
+.Prerequisites
+
+* Access to the {product-title} web console.
+* The Elasticsearch Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Operator Management*.
+
+. Click the *Project* menu, and then click *openshift-operators*.
+
+. Click the *elasticsearch-operator* Options menu {kebab}, and then click *Remove Subscription*.
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
+
+[id="ossm-remove-operator-jaeger_{context}"]
+== Removing the Jaeger Operator
+Follow this procedure to remove the Jaeger Operator.
+
+.Prerequisites
+
+* Access to the {product-title} web console.
+* The Jaeger Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Operator Management*.
+
+. Click the *Project* menu, and then click *openshift-operators*.
+
+. Click the *jaegeroperator* Options menu {kebab}, and then click *Remove Subscription*.
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
+[id="ossm-remove-operator-kiali_{context}"]
+== Removing the Jaeger Operator
+Follow this procedure to remove the Kiali Operator.
+
+.Prerequisites
+
+* Access to the {product-title} web console.
+* The Kiali Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Operator Management*.
+
+. Click the *Project* menu, and then click *openshift-operators*.
+
+. Click the *kiali-ossm* Options menu {kebab}, and then click *Remove Subscription*.
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
+[id="ossm-remove-cleanup_{context}"]
+== Clean up Operator resources
+Follow this procedure to manually remove resources left behind after removing the {ProductName} Operator by using the OperatorHub interface.
+
+.Prerequisites
+
+* An account with cluster administration access.
+* Access to the {product-title} Command-line Interface (CLI) also known as `oc`.
+
+.Procedure
+
+. Log into the {product-title} CLI as a cluster administrator.
+
+. Run the following commands to clean up resources after uninstalling the Operators:
++
+[NOTE]
+====
+Replace `<operator-project>` with the name of the project where the {ProductName} Operator was installed. This is typically `openshift-operators`.
+====
++
+----
+$ oc delete validatingwebhookconfiguration/<operator-project>.servicemesh-resources.maistra.io
+$ oc delete -n <operator-project> daemonset/istio-node
+$ oc delete clusterrole/istio-admin
+$ oc get crds -o name | grep '.*\.istio\.io' |xargs -r -n 1 oc delete
```

Comment:
For consistency I'd include a space between `|` and `xargs`, but that's stylistic and `bash` won't care.

Reply:
Fixed.